_woc/Welcome to my WoC blog.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Text Sentiment Analysis Starter Blog"
```

Comment:
Change this to a suitable title: My Winter of Code Story/Jumping into Text Sentiment Analysis or something innovative!